css/style.css

```diff
@@ -55,6 +48,7 @@ html {
   text-transform: uppercase;
   font-size: 3.6rem;
   opacity: 90%;
+  margin-right: 22rem;
```

Comment:
Consider `margin-right: auto;`

Reply:
did not solve the problem
![2022-12-26_10-53-19](https://user-images.githubusercontent.com/110885647/209521211-bbb0a1e1-95d3-46e6-87d5-9b5bc06d805c.png)

Reply:
I will look for a solution